package.json

```diff
@@ -20,6 +20,7 @@
     "react-dom": "18.2.0",
     "react-flatpickr": "^3.10.13",
     "react-phone-input-2": "^2.15.1",
+    "react-quill": "^2.0.0",
```

Comment:
Hi Alex, if i may ask, i see that react quill is already listed as a dependency on staging, why was it a change here? :D